src/core/pyproxy.js

```diff
@@ -348,26 +348,38 @@ class PyProxyClass {
   }
   /**
    * Converts the ``PyProxy`` into a Javascript object as best as possible. By
-   * default does a deep conversion, if a shallow conversion is desired, you
-   * can use ``proxy.toJs({depth : 1})``.
-   * See :ref:`Explicit Conversion of PyProxy
+   * default does a deep conversion, if a shallow conversion is desired, you can
+   * use ``proxy.toJs({depth : 1})``. See :ref:`Explicit Conversion of PyProxy
    * <type-translations-pyproxy-to-js>` for more info.
    *
    * @param {object} options
-   * @param {number} options.depth How many layers deep to perform the conversion.
-   * Defaults to infinite.
+   * @param {number} [options.depth] How many layers deep to perform the
+   * conversion. Defaults to infinite.
+   * @param {array} [options.pyproxies] If provided, ``toJs`` will store all
+   * PyProxies created in this list. This allows you to easily destroy all the
```

Comment:
It would be also worth mentioning that's it's expected to pass an empty array.

Reply:
To be clear there is a perfectly valid use case for passing nonempty arrays. Namely you may want to batch deallocate:
```js
let a = [];
let x1 = px1.toJs({ pyproxies : a});
let x2 = px2.toJs({ pyproxies : a});
let x3 = px3.toJs({ pyproxies : a});
// Do stuff
// ...
// Reclaim all proxies
for(let p of a){
   p.destroy();
}
``
I will add extra explanation. It would be useful to add a similar option to `runPython`, but it may require more careful consideration first.